_sass/minima/_layout.scss

```diff
@@ -8,15 +8,6 @@
 
 .page-content .wrapper {
   padding-bottom: 3em;
-
-  // This is to style non-english page title, until .hero is added
```

Comment:
Moved this to it's own, scoped style file to prevent issues it was causing elsewhere.